server/cluster.go

```diff
@@ -120,6 +120,20 @@ func (c *RaftCluster) isRunning() bool {
 	return c.running
 }
 
+// GetHotRegions gets all hot regions status
+func (s *Server) GetHotRegions() map[uint64]*StoreHotRegions {
+	schedule, ok := s.cluster.coordinator.schedulers[hotRegionScheduleName]
```

Comment:
um...It's for http API, just read and do not care about the data is up to date. I think no need to do that.